catalog/imaging/catalog-imaging-service-impl/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Connexta, LLC
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+ -->
+
+<blueprint default-activation="lazy"
+           xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="
+              http://www.osgi.org/xmlns/blueprint/v1.0.0 http://www.osgi.org/xmlns/blueprint/v1.0.0/blueprint.xsd">
+
+    <bean id="chipService" class="com.connexta.alliance.imaging.chip.service.impl.ChipServiceImpl"/>
+
+    <service interface="com.connexta.alliance.imaging.chip.service.api.ChipService"
+             ref="chipService">
+        <service-properties>
+            <entry key="id" value="chipService"/>
+            <entry key="shortname" value="chipService"/>
+            <entry key="title" value="Chip image..."/>
+            <entry key="description"
+                   value="Supplies image chipping services to DDF."/>
```

Comment:
This service isn't NITF/NSIF specific.  You pass it the image and 2 polygons, so the source of the image isn't relevant.  It would be reusable to chip things like GeoJpeg or a frame of video.